index.html

```diff
@@ -4,28 +4,37 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta charset="UTF-8">
     <title>Flickr-Poc</title>
-    <link rel="stylesheet" type="text/css" href="css/style.css">
     <link rel="stylesheet" type="text/css" href="bower_components/lightbox2/dist/css/lightbox.css">
+    <link rel="stylesheet" type="text/css" href="bower_components/bootstrap/dist/css/bootstrap.min.css">
+
+    <link rel="stylesheet" type="text/css" href="css/style.css">
 </head>
 <body>
-<div class="container">
-
-    <h1>Flickr-Poc</h1>
-
-    <h3>Gallery</h3>
-
-    <div class="image-slider-wrapper">
-
-        <div class="image-slider-wrapper">
-            <ul id="list">
+    <div class="navbar-wrapper">
+        <div class="container-fluid">
+            <nav>
+                <div class="navbar navbar-static-top">
+                    <div class="container">
+                        <div class="navbar-header">
+                            <a href="#" class="navbar-brand" id="logo">Flickr-Poc</a>
+                        </div>
+                    </div>
+                </div>
+            </nav>
+        </div>
+    </div>
 
-            </ul>
+    <div class="container-responsive">
```

Comment:
cool, and because of the col-offset we will still be able to have a nice margin around it. Thanks.